### PR TITLE
Make wx backend working as same as qt backend.

### PR DESCRIPTION
**Pull request by walker8088**

### CODE REVIEW

File: vispy/scene/cameras/perspective.py [hunk 1]
```diff
@@ -93,7 +93,10 @@ def _set_range(self, init):
         # Get window size (and store factor now to sync with resizing)
         w, h = self._viewbox.size
         w, h = float(w), float(h)
-
+        
+        if (w == 0) or (h == 0):
+                return
```
Comment (Eric89GXL) [inline vispy/scene/cameras/perspective.py]: this is over-indented

File: vispy/scene/cameras/perspective.py [hunk 2]
```diff
@@ -137,6 +140,10 @@ def _update_transform(self, event=None):
 
         # Correct for window size
         w, h = self._viewbox.size
+        
+        if (w == 0) or (h == 0):
+                return
```
Comment (Eric89GXL) [inline vispy/scene/cameras/perspective.py]: same here

File: vispy/scene/cameras/perspective.py (hunk 1 above)
Comment (walker8088) [inline vispy/scene/cameras/perspective.py]: fixed. please check it. again

**Comment by Eric89GXL** [comment]
Thanks @walker8088

**Pull request merged**